e2e/cypress/integration/account_settings/display/timezone_display_mode_spec.js

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @account_setting
+
+import moment from 'moment-timezone';
```

Comment:
Please explicitly include `moment-timezone` in package.json. It probably works since other package have it as its own dependency.

Reply:
Added to package.json